et --hard HEAD~1q

```diff
@@ -0,0 +1,95 @@
+[33mcommit 88670d94263b30e5792852275db7a9b7893fddc8[m[33m ([m[1;36mHEAD -> [m[1;32mmain[m[33m, [m[1;31morigin/springBootExerciseNr1[m[33m, [m[1;31morigin/main[m[33m, [m[1;32mspringBootExerciseNr1[m[33m, [m[1;32msecurityexercise[m[33m)[m
```

Comment:
this file probably got committed accidentally?

Reply:
yes, it was